diff --git a/node_modules/next/dist/server/next-server.js b/node_modules/next/dist/server/next-server.js
index 259932d..c0d4859 100644
--- a/node_modules/next/dist/server/next-server.js
+++ b/node_modules/next/dist/server/next-server.js
@@ -1245,7 +1245,7 @@ class NextNodeServer extends _baseserver.default {
                 });
             } finally{
                 if (hasRequestBody) {
-                    requestData.body.finalize();
+                    await requestData.body.finalize();
                 }
             }
         } else {
